### ISSUE TITLE
omr acceptance builds launching with the same omr sha

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#4813]
A number of omr acceptance builds launched with the same omr sha.
https://ci.eclipse.org/openj9/view/Pipelines/job/Pipeline-OMR-Acceptance/

Look at 1194 - 1199

**Comment by pshipton** [comment]
@jdekonin @AdamBrousseau

**Comment by pshipton** [comment]
Also 1186 - 1190 and 1191 - 1192. Disabled the build for now.

**Comment by pshipton** [comment]
Disabled the Mirror-OMR-to-OpenJ9-OMR Pipeline as well since it runs and fails every 15 min, posting the failure to jenkins.

**Comment by AdamBrousseau** [comment]
No idea, added some debug output to the script and now it seems to be working as expected. I'll let you re-enable when the cmake compile is resolved.

**Comment by pshipton** [comment]
Unfortunately so many builds were created that some nightly test failure results got purged from the history (which is only 10 builds).

**Comment by pshipton** [comment]
The Pipeline-OMR-Acceptance was already enabled, and I've enabled Mirror-OMR-to-OpenJ9-OMR as well. We can monitor any new builds started.

**Comment by pshipton** [comment]
@AdamBrousseau it is still happening, see 1202 - 1203.

Disabling Mirror-OMR-to-OpenJ9-OMR again.

**Comment by jdekonin** [comment]
I think for the last one its a matter of sha length. omr_sha=1537cc08 was generated from 1202 via 47742 (Mirror job), omr_sha=1537cc0 was generated from 1203 via 47743 (Mirror job).

The acceptance build launch is determined on exact match determine within the OMR mirror script, so I think the last one work as expected, or rather how the code is written.
https://github.com/eclipse/openj9/blob/1660a2f706ac744f4fd5364c97d80e3a9769cb8e/buildenv/jenkins/jobs/infrastructure/omrMirror#L70

We might want change that as different versions of git return a different length for rev-parse.

ub16-ppcle-4 (added to be a worker yesterday), has v2.7.4 where rev-parse returns 7 characters.
proxy-worker I don't have access to verify the git version, but rev-parse returns 8 characters.

I disabled `ub16-ppcle-4` and reran the Mirror and it appears to work as expected, keeping in mind I needed to launch twice.  Once to get the correct 8 char sha as a baseline build, and the second to verify the downstream acceptance build wouldn't be launched which it was not.
https://ci.eclipse.org/openj9/job/Mirror-OMR-to-OpenJ9-OMR/47744/console
https://ci.eclipse.org/openj9/job/Mirror-OMR-to-OpenJ9-OMR/47745/console

Fix should be to fix the length of the rev-parse.  `git rev-parse --short=8 HEAD`
